sdk/cosmos/sdk/src/test/java/com/azure/data/cosmos/rx/VeryLargeDocumentQueryTest.java

```diff
@@ -32,10 +12,7 @@
 import com.azure.data.cosmos.FeedOptions;
 import com.azure.data.cosmos.FeedResponse;
 import org.apache.commons.lang3.StringUtils;
-import org.testng.annotations.AfterClass;
-import org.testng.annotations.BeforeClass;
-import org.testng.annotations.Factory;
-import org.testng.annotations.Test;
+import org.testng.annotations.*;
```

Comment:
@JonathanGiles what's the suggested code style for import?
should we use wildcard import?

Reply:
This should not be wildcard, updating it back.

Reply:
reverted.